### DISCUSSION TITLE
Atom isn't installing on debian 10

### DISCUSSION THREAD

**Discussion by Manga-Man** [atom/atom#22625] [Q&A]
I tried installing the .deb file but it didn't work...
![image](https://user-images.githubusercontent.com/72046004/122804167-65c32e80-d27c-11eb-88a5-8ef6671fca66.png)
![image](https://user-images.githubusercontent.com/72046004/122804224-7b385880-d27c-11eb-8207-0662e0794cad.png)
I tried to build from the source but It still didn't work.
![image](https://user-images.githubusercontent.com/72046004/122804543-e5e99400-d27c-11eb-8734-8794e1f1f192.png)
I'm not that Linux smart...I rlly need help.
my system:
![image](https://user-images.githubusercontent.com/72046004/122804814-306b1080-d27d-11eb-8a8a-f860607189c0.png)

**Comment by saabman** [comment]
Im not real good in the Linux domain either but I successfully installed ATOM on Debian 10 by using the instructions on this page
https://flight-manual.atom.io/getting-started/sections/installing-atom/
